Documentation: Update Truth Data instructions (#2154)
dtcenter/METplus: docs/Contributors_Guide/add_use_case.rst

    - "met_tool_wrapper:0-54"

Update the Truth Data
---------------------

> The addition of a new use case results in new output data. When this happens,

Reviewer: George, I realize that this section is about adding a new use case. But I do think its worth noting in some way that changes to existing use cases and bugfixes can also change the output. Feel free to use the wording below... or re-word it however you see fit.

```suggestion
The addition of a new use case, changes to an existing use case, or software bug fixes can result in new or modified output data. When this happens,
```

responsible for the changes in the truth data can be referenced to easily
track where differences occurred.

A GitHub Action workflow is available to handle this step.

* Navigate to https://github.com/dtcenter/METplus/actions/workflows/update_truth.yml
  or from the METplus GitHub page, click on the Actions tab,
  then click on "Update Truth Data" under menu on the left.
* Click on the "Run workflow" button on the right.
* Click on the Branch pull down and select "develop" unless you are updating
  the truth data for a bugfix on a main_vX.Y branch.
* Enter the pull request numbers that warranted the update.
  Include the '#' symbol before the number to create a link to the PR.
  PRs from a repository other than METplus should include

>   the repository name before '#' symbol.

Reviewer: ```suggestion
  'dtcenter/' and the repository name before '#' symbol (e.g. dtcenter/MET#2000).
```

* Enter a brief summary of the changes.
  Developers can navigate to the PRs for more information.

.. figure:: figure/update_truth_data.png

* Click the "Run workflow" button.
* A new workflow run should appear at the top of the list and complete quickly.
* Click on the "Pull Requests" tab.
  A new pull request should have been created with the information that
  was entered. Click on the new pull request.
* Verify that the information in this pull request is correct.
  If the "develop" branch was selected in the "Run workflow" menu,
  then the pull request should show **develop-ref <- develop**.

> * Add the appropriate project and milestone values on the right hand side.

Reviewer: ```suggestion
* Add the appropriate project, development cycle, and milestone values on the right hand side.
```
